hc-tree-node.html

```diff
@@ -124,6 +125,11 @@
         __init: {
           type: Boolean,
           value: true
+        },
+        /* The value to disable the node*/
+        disabled: {
```

Comment:
add the `reflectToAttribute: true` flag here as well so it gets placed in the DOM for semantics without having to use `$=`.